docs/reference/setup/advanced-configuration.asciidoc

```diff
@@ -117,6 +117,13 @@ threshold, check the {es} log for an entry like this:
 ----
 heap size [1.9gb], compressed ordinary object pointers [true]
 ----
++
+Or check the <<cluster-nodes-info,nodes info API's>> `nodes.{name}.jvm.using_compressed_ordinary_object_pointers` value:
```

Comment:
What is the correct method for referring to a field returned in the JSON response?
I didn't know so I put in `{name}` as I wasn't sure what else to call that level.

```json
{
  "_nodes" : {
    "total" : 1,
    "successful" : 1,
    "failed" : 0
  },
  "cluster_name" : "...",
  "nodes" : {
    "iIo93x-eSIyrvodrPuOOHw" : {
      "name" : "instance-0000000016",
...
      "version" : "7.17.3",
...
      "roles" : [...],
      "attributes" : {...},
      "jvm" : {
...
        "mem" : {...},
        "gc_collectors" : [...],
        "memory_pools" : [...],
        "using_compressed_ordinary_object_pointers" : "true",
        "input_arguments" : [...]
      }
    }
  }
}
```

Reply:
At minimum, I should probably update it to be `{id}` instead of `{name}`.
```suggestion
Or check the <<cluster-nodes-info,nodes info API's>> `nodes.{id}.jvm.using_compressed_ordinary_object_pointers` value:
```